lib/xsl/jats-to-html.xsl

```diff
@@ -1745,6 +1745,12 @@
         </sub>
     </xsl:template>
 
+    <xsl:template match="disp-quote">
+        <blockquote class="disp-quote">

```

Comment:
This doesn't seem to be working. I'm not sure why but the follow but we had a similar issue with the math element and the following should work:

```
    <xsl:template match="disp-quote">
        <xsl:text disable-output-escaping="yes">&lt;blockquote class="disp-quote"&gt;</xsl:text>
            <xsl:apply-templates/>
        <xsl:text disable-output-escaping="yes">&lt;/blockquote&gt;</xsl:text>
    </xsl:template>
```